arctic/chunkstore/chunkstore.py

```diff
@@ -0,0 +1,376 @@
+import logging
+import pymongo
+import numpy as np
+import bson
+
+from bson.binary import Binary
+from pandas import Series, DataFrame
+
+from ..store._version_store_utils import checksum
+from ..decorators import mongo_retry
+from .._util import indent
+from ..serialization.pandas_serializer import (serialize, deserialize,
+                                               DataFrameSerializer,
+                                               SeriesSerializer,
+                                               PandasSerializer)
+from .date_chunker import DateChunker
+from ..exceptions import UnhandledDtypeException, NoDataFoundException
+from .._compression import compress_array, decompress
+
+
+logger = logging.getLogger(__name__)
+
+CHUNK_STORE_TYPE = 'ChunkStoreV1'
+
+
+class ChunkStore(object):
+    STRING_MAX = 16
+
+    @classmethod
+    def initialize_library(cls, arctic_lib, **kwargs):
+        ChunkStore(arctic_lib)._ensure_index()
+
+    @mongo_retry
+    def _ensure_index(self):
+        collection = self._collection.symbols
+        collection.create_index([("symbol", pymongo.ASCENDING),
+                                 ("_id", pymongo.DESCENDING)],
```

Comment:
Just wondering why `_id` is useful in this index? It looks like, at the moment there's only ever one instance of a symbol in the symbols collection at any time?